newapi/ooniapi/auth.py

```diff
@@ -71,7 +73,11 @@ def jerror(msg, code=400):
 
 def create_jwt(payload: dict) -> str:
     key = current_app.config["JWT_ENCRYPTION_KEY"]
-    return jwt.encode(payload, key, algorithm="HS256").decode()
+    token = jwt.encode(payload, key, algorithm="HS256")
+    if isinstance(token, bytes):
+        return token.decode()
+    else:
```

Comment:
nitpick, you don't need the else statement here.

Reply:
It's just for readability and safety - to stress that we are running either line 78 or 80

Reply:
I guess this is a matter of code style. I personally tend to prefer dropping else statements when they are not needed as I find it more readable.

Also, why is it that we need to handle cases in which the token might be in bytes or not? From what I see `jwt.encode` returns an `str`, in which cases is it going to be bytes?